This patch uses utimes instead of futimes for uclibc
since futimes is not available

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: git/examples/solv.c
===================================================================
--- git.orig/examples/solv.c	2012-06-01 12:06:22.041552848 -0700
+++ git/examples/solv.c	2012-06-01 12:49:17.417677449 -0700
@@ -1027,7 +1027,8 @@
   int flags;
 
   cinfo = repo->appdata;
-  if (!(fp = fopen(calccachepath(repo, repoext), "r")))
+  const char* fname = calccachepath(repo, repoext);
+  if (!(fp = fopen(fname, "r")))
     return 0;
   if (fseek(fp, -sizeof(mycookie), SEEK_END) || fread(mycookie, sizeof(mycookie), 1, fp) != 1)
     {
@@ -1068,7 +1069,11 @@
       memcpy(cinfo->extcookie, myextcookie, sizeof(myextcookie));
     }
   if (mark)
+#ifdef __UCLIBC__
+    utimes(fname, 0);	/* try to set modification time */
+#else
     futimes(fileno(fp), 0);	/* try to set modification time */
+#endif
   fclose(fp);
   return 1;
 }
